test/e2e/network/service.go

```diff
@@ -187,33 +178,33 @@ var _ = SIGDescribe("Services", func() {
 		// repacking functionality is intentionally not tested here - it's better to test it in an integration test.
 		serviceName := "multi-endpoint-test"
 		ns := f.Namespace.Name
-		jig := e2eservice.NewTestJig(cs, serviceName)
+		jig := e2eservice.NewTestJig(cs, ns, serviceName)
 
 		defer func() {
 			err := cs.CoreV1().Services(ns).Delete(serviceName, nil)
 			framework.ExpectNoError(err, "failed to delete service: %s in namespace: %s", serviceName, ns)
 		}()
 
-		labels := map[string]string{"foo": "bar"}
-
 		svc1port := "svc1"
 		svc2port := "svc2"
 
 		ginkgo.By("creating service " + serviceName + " in namespace " + ns)
-		ports := []v1.ServicePort{
-			{
-				Name:       "portname1",
-				Port:       80,
-				TargetPort: intstr.FromString(svc1port),
-			},
-			{
-				Name:       "portname2",
-				Port:       81,
-				TargetPort: intstr.FromString(svc2port),
-			},
-		}
-		_, err := jig.CreateServiceWithServicePort(labels, ns, ports)
-		framework.ExpectNoError(err, "failed to create service with ServicePorts in namespace: %s", ns)
+		_, err := jig.CreateTCPService(func(service *v1.Service) {
```

Comment:
How are we specifying the labels here (`foo=bar`)?
it looks like they are not being set.